hardhat.config.ts

```diff
@@ -61,7 +66,14 @@ const config: HardhatUserConfig = {
         // },
         fuji: {
             url: "https://api.avax-test.network/ext/bc/C/rpc",
-            accounts: [env.DEV_WALLET_PRIVATE_KEY],
+            accounts: [
+                env.DEV_WALLET_PRIVATE_KEY,
+                env.NEXUS_MASTER_PRIVATE_KEY,
+                env.PROFILE_NFT_COUNSELOR_PRIVATE_KEY,
+                env.TAVERN_OWNER_PRIVATE_KEY,
```

Comment:
this should be updated to one address 
GUILD_MASTER_KEY
GUILD_MASTER_PUBLIC_KEY

Reply:
updated to guild master